variable/variable.test.functions

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
```

Comment:
Yep. And it works better on platforms like debian, where /bin/sh => /bin/dash ...